From 6e5543c447dab43370303ade4c4f91c2a263a5ea Mon Sep 17 00:00:00 2001
From: Paul Bovbel <pbovbel@locusrobotics.com>
Date: Wed, 29 Nov 2017 23:53:00 -0500
Subject: [PATCH] Sync patches with lunar

---
 CMakeLists.txt | 22 ++++++++++++++++++++++
 package.xml    |  3 ++-
 2 files changed, 24 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d79ee4c..1218030 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,6 +20,14 @@ include(OptionRequires)
 include(CTest)
 include(CheckIncludeFileCXX)
 
+set(VRPN_BUILD_CLIENT_LIBRARY TRUE)
+set(VRPN_BUILD_JAVA FALSE)
+set(VRPN_BUILD_PYTHON FALSE)
+set(VRPN_BUILD_CLIENTS FALSE)
+set(VRPN_BUILD_SERVERS FALSE)
+set(VRPN_BUILD_SERVER_LIBRARY FALSE)
+set(BUILD_TESTING FALSE)
+
 if((NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR) OR VRPN_SUBPROJECT_BUILD)
 	# If you're using this as a subproject and want things installed, set VRPN_INSTALL TRUE.
 	set(SUBPROJECT TRUE)
@@ -1369,6 +1377,20 @@ if(VRPN_BUILD_CLIENT_LIBRARY)
 			LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT clientsdk
 			ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT clientsdk
 			PUBLIC_HEADER DESTINATION include COMPONENT clientsdk)
+
+		# Install catkin package.xml
+		install(FILES package.xml DESTINATION share/vrpn)
+
+		install(
+			FILES cmake/FindVRPN.cmake
+			DESTINATION share/vrpn/cmake
+			RENAME VRPNConfig.cmake)
+
+		install(
+			FILES cmake/Findquatlib.cmake
+			DESTINATION share/quatlib/cmake
+			RENAME quatlibConfig.cmake)
+
 	endif()
 	add_cppcheck(vrpn STYLE UNUSED_FUNCTIONS)
 endif()
diff --git a/package.xml b/package.xml
index 6dd130b..f13b18e 100644
--- a/package.xml
+++ b/package.xml
@@ -8,7 +8,8 @@
   <license>BSL1.0</license>
   <url type="website">https://github.com/vrpn/vrpn/wiki</url>
 
-  <maintainer email="paul@bovbel.com">Paul Bovbel</maintainer>
+  <!-- Maintainer of the ROS packaged release. -->
+  <maintainer email="pbovbel@clearpathrobotics.com">Paul Bovbel</maintainer>
 
   <buildtool_depend>cmake</buildtool_depend>
 
-- 
2.17.0

